index.html

```diff
@@ -90,7 +90,234 @@ <h1 class="primary-text-header">I’m Steven Levoyer to see you!</h1>
     </header>
     <!--End-Header-->
     <!--Main-->
+    <main>
+        <!--Works-->
+        <section aria-label="Section of works">
+            <div class="work">
+                <div class="card-works">
+                    <img class="snapshoot" src="./img/Snapshoot-1.svg">
+                    <div class="left-block">
+                        <div class="primary-text-work">
+                            <h2 class="project-title">Tonic</h2>
+                            <div class="frame-2">
+                                <div class="client">
+                                    <p class="text-p">CANOPY</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <p class="text-p">Back End Dev</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year">
+                                    <p class="text-p">2015</p>
+                                </div>
+                            </div>
+                        </div>
```

Comment:
- [ ] Kindly use an indentation of two spaces in all your documents. This is a convention and also best practice.

Reply:
Hi, @Kamba56 
I don't understand what you mean by sangria.
Is this for the texts in the labels or for the labels?